buku.py

```diff
@@ -329,6 +329,95 @@ def decrypt_file(iterations, dbfile=None):
             sys.exit(1)
 
 
+def import_md(filepath, newtag):
+    '''Parse bookmark markdown file
+
+    :param filepath: Markdown file
+    :param newtag: New tag
+    :return: a tuple containing parsed result
+    '''
+    with open(filepath, mode='r', encoding='utf-8') as infp:
+        for line in infp:
+            # Supported markdown format: [title](url)
+            # Find position of title end, url start delimiter combo
+            index = line.find('](')
+            if index != -1:
+                # Find title start delimiter
+                title_start_delim = line[:index].find('[')
+                # Reverse find the url end delimiter
+                url_end_delim = line[index + 2:].rfind(')')
+
+                if title_start_delim != -1 and url_end_delim > 0:
+                    # Parse title
+                    title = line[title_start_delim + 1:index]
+                    # Parse url
+                    url = line[index + 2:index + 2 + url_end_delim]
+                    if (is_nongeneric_url(url)):
+                        continue
+
+                    yield (
```

Comment:
Please use till end of line (limit 80 chars). We follow that convention almost everywhere.